src/sentry/templates/sentry/toolbar/iframe.html

```diff
@@ -1,15 +1,75 @@
+{% comment %}
+Template returned for authenticated, valid requests to /iframe. The iframe serves as a proxy for Sentry API requests.
+Required context variables:
+- referrer:         string. HTTP header from the request object.
+{% endcomment %}
 {% load sentry_helpers %}
 {% load sentry_assets %}
 <!DOCTYPE html>
-<html>
-<head>
-  <title></title>
-</head>
-<body>
-  {% script %}
-  <script>
-  console.log("Hello World!");
-  </script>
-  {% endscript %}
-</body>
+<html lang="en">
+  <head>
+    <title>Sentry DevToolbar iFrame</title>
+    <link rel="icon" type="image/png" href="{% absolute_asset_url "sentry" "images/favicon.png" %}">
+  </head>
+  <body>
+    <script>
+      const referrer = "{{ referrer|escapejs }}";
+
+      function log(...args) {
+        console.log('iframe:', ...args);
+      }
+
+      log('Init', {referrer});
+
+      const {port1, port2} = new MessageChannel();
+
+      const messageDispatch = {
+        'log': log,
+        'fetch': async (url, init) => {
+          const response =  await fetch(url, {
+            ...init,
+            headers: {
+              ...init.headers,
+              // For mock/test purposes we're fetching the accessToken and injecting it
+              'Authorization': `Bearer ${localStorage.getItem('accessToken')}`
+            }
```

Comment:
We can remove this because in prod it'll just automatically use the cookie

```suggestion
```

Reply:
Do we still need `...init.headers`?

Reply:
it's got this above so should be spreading headers, unmodified, into there:
```
const response =  await fetch(url, {
  ...init,
```